### PR TITLE
Ben/lmb 595 bug fracties aanmaken

### PR DESCRIPTION
**Pull request by bfidlers**
## Description

It is no longer possible to add fracties if the selected bestuursperiode has no bestuursorganen. This is disabled because if this is possible these fracties are not really useful, they aren't shown, because fracties are only fetched based on bestuursorganen and these fracties would never be added to a bestuursorgaan, even if new bestuursorganen were added, so they have no point at all. 
This is disabled in two places. In the organen page: 
Screenshot 2024-07-15 at 15 16 24
And in the bestuursperiode selector of the fractie page.

## How to test

Go to the organen page and see what happens with the beheer fracties button depending on which bestuursperiode is selected. Also if you go to the beheer fracties page, the bestuursperiode selector only shows bestuursperiods for which it makes sense to edit fracties (they should have bestuursorganen and they should not have an active prepare legislature page.

### CODE REVIEW

File: app/routes/organen/fracties.js [hunk 1]
```diff
@@ -25,9 +28,35 @@ export default class FractiesRoute extends Route {
 
     const bestuursPeriods = await this.store.query('bestuursperiode', {
       sort: 'label',
+      'filter[:has:heeft-bestuursorganen-in-tijd]': 'true',
```
Comment (JonasVanHoof) [inline app/routes/organen/fracties.js:31]: NITPICK: Does true need to be a string?

File: app/routes/organen/fracties.js [hunk 2]
```diff
@@ -25,9 +28,35 @@ export default class FractiesRoute extends Route {
...
+          }
+          return {
+            value: period,
+            valid,
+          };
+        })
+      )
+    )
+      .filter((v) => v.valid)
+      .map((data) => data.value);
```
Comment (JonasVanHoof) [inline app/routes/organen/fracties.js:56]: This is great 😁

**Review by JonasVanHoof — APPROVED** [review]
Looks good! Tested the frontend to 👌🏻

File: app/routes/organen/fracties.js (hunk 1 above)
Comment (bfidlers) [reply app/routes/organen/fracties.js]: indeed it doesn't

File: app/routes/organen/fracties.js (hunk 2 above)
Comment (bfidlers) [reply app/routes/organen/fracties.js]: Improved it a little bit

Comment (JonasVanHoof) [reply app/routes/organen/fracties.js]: Alright, indeed a cleaner piece of code

**Comment by JonasVanHoof** [comment]
I was thinking should we show a message to the user why these options are disabled? I was just thinking that we disable the fractie beheer button for two reasons now 🤔

**Comment by bfidlers** [comment]
> I was thinking should we show a message to the user why these options are disabled? I was just thinking that we disable the fractie beheer button for two reasons now 🤔

Yeah might be getting a bit confusing for the user, will mention it in the standup so Tim can look into it.

**Pull request merged**